src/components/PressGridview.js

```diff
@@ -0,0 +1,106 @@
+import { shuffleArray } from '../utils.js';
+import { NEWS_PRESS_NUMBERS_PER_PAGE, PAGE_MIN_NUMBER, PAGE_MAX_NUMBER } from '../constants.js';
+
+export default function PressGridView({ $target, initialState }) {
+  const $section = document.createElement('section');
+  $section.classList.add('news-press-display');
+
+  $target.appendChild($section);
+
+  this.state = initialState;
+
+  this.setState = nextState => {
+    this.state = nextState;
+    this.render();
+  };
+
+  const fetchNewsPressData = () => {
+    fetch('../data/press-info.json')
+      .then(response => response.json())
+      .then(json => {
+        const shufffledNewsPressData = shuffleArray(json);
+        this.setState({
+          ...this.state,
+          newsPressData: shufffledNewsPressData,
+        });
+      });
+  };
+
+  const initNewsPressItems = () => {
+    const $ul = $section.querySelector('ul');
+    $ul.innerHTML = '';
+
+    const { page, newsPressData } = this.state;
+
+    const startIndex = NEWS_PRESS_NUMBERS_PER_PAGE * (page - 1);
+    const endIndex = startIndex + 23;
+    const currentNewsPressData = newsPressData.slice(startIndex, endIndex + 1);
+
+    currentNewsPressData.forEach(newsPressItem => {
+      const $li = document.createElement('li');
+      const $img = document.createElement('img');
+      const { logo } = newsPressItem;
+
+      $img.src = logo;
+      $img.classList.add('press-logo');
+
+      $li.appendChild($img);
+      $li.classList.add('news-press-item');
+
+      $ul.appendChild($li);
+    });
+  };
```

Comment:
```suggestion
  const initNewsPressItems = () => {
    const $ul = $section.querySelector('ul');
    $ul.innerHTML = '';

    const { page, newsPressData } = this.state;

    const startIndex = NEWS_PRESS_NUMBERS_PER_PAGE * (page - 1);
    const endIndex = startIndex + 23;
    const currentNewsPressData = newsPressData.slice(startIndex, endIndex + 1);
    const fragment = document.createDocumentFragment();
    currentNewsPressData.forEach(newsPressItem => {
      const $li = document.createElement('li');
      const $img = document.createElement('img');
      const { logo } = newsPressItem;

      $img.src = logo;
      $img.classList.add('press-logo');

      $li.appendChild($img);
      $li.classList.add('news-press-item');
      fragment.appendChild($li);
    });
    $ul.appendChild(fragment);
  };
```
로 작성하는게 마지막에 한번만 DOM에 추가해서 성능이 살짝 좋아질 것 같아요 빈 태그 뭐 있나 싶어서 검색했는데 [createDocumentFragment()](https://iamawebdeveloper.tistory.com/64) 라는게 있다고해요~